handsontable/src/3rdparty/walkontable/src/table.js

```diff
@@ -767,6 +768,7 @@ class Table {
     let rowCount = this.TBODY.childNodes.length;
     const expectedTableHeight = rowCount * this.wtSettings.getSetting('defaultRowHeight');
     const actualTableHeight = innerHeight(this.TBODY) - 1;
+    const rowHeightFn = StylesManager.CELL_BOX_SIZING === 'border-box' ? outerHeight : innerHeight;
```

Comment:
The same as the previous comment, it should be configurable throughout the Walkontable settings.